### ISSUE TITLE
Shift USB location to line up with hole in Z board

### ISSUE THREAD

**Issue by jbwillis** [PyCubed-Mini/OpenMV_Camera#1]
USB connector location needs to be moved to line up with the access hole in the Z board. The USB connector also needs to protrude through the hole in order to properly attach a cable to it. 

- [ ] Determine location of Z board hole with respect to camera board
- [ ] Move the connector to line up with the Z board hole horizontally
- [ ] Move the connector to protrude through the Z board hole
- [ ] Update 3D model and check clearances.

**Comment by NeilKhera** [comment]
It is okay to change the board outline of the camera board to be exactly the same as the mainboard? This would ensure that the usb port is at exactly the same spot. At the moment the camera board is slightly shorter than the main board which places the connector further back from the Z board

**Comment by jbwillis** [comment]
Yes, but I’d restrict the change to be only in the area around the USB port
to avoid potential interference with the Z Board.
